Rewards/Classes/Rewards.swift

```diff
@@ -47,21 +48,24 @@ public class Rewards{
     public static let account = AccountService()
     
     /// An instance of `CaptureService` for handling data capture functionalities.
-    public static let capture = CaptureService()
+    public static var capture = CaptureService.init()
     
     /// An instance of `LicenseService` for managing data licenses.
     public static let license = LicenseService()
     
+    public static var configuration: Configuration? = nil
```

Comment:
When is this configuration passed to Capture Receipt?

Reply:
Should be in the config, i'll see this